src/components/Reports/FourteenMonthReports/Layout/Table/helpers.test.ts

```diff
@@ -20,10 +21,10 @@ const mockContact = (
 describe('extractSortKey', () => {
   it('extracts string values from the contact', () => {
     const contact = mockContact({
-      status: 'Partner',
+      status: StatusEnum.PartnerFinancial,
     });
 
-    expect(extractSortKey(contact, 'status')).toBe('Partner');
+    expect(extractSortKey(contact, 'status')).toBe('PARTNER_FINANCIAL');
```

Comment:
Could this just be:
```suggestion
.toBe(StatusEnum.PartnerFinancial);
```

Reply:
Good point. The point of that test is just that it pulls out a string field from the contact, so I kind of wanted to `toBe` to be a string literal. I think it makes more sense to pick a different field for that test.